Add in all names for the batch9 contact list

acchiang88 opened the pull request.

ehtibareziz committed:
this is a new salam

ehtibareziz committed:
SALAMLAR bolsum

ehtibareziz committed:
I add my email address..

AnnaZhao0209 committed:
Merge branch 'master' into add-all-names

Jack-Vu reviewed (changes requested):
Just make changes to the contacts.txt. In the branch you are working on you are making commits to an index.txt and this document will be send up when pushing. Delete that file or move it and then commit the changes to just the contacts file.

acchiang88 commented:
Hello Jack,

You means just change everything in the contacts.txt not in the index.txt?

Thank you!!

Addie

Jack Vu ***@***.***> 於 2023年2月13日 週一 上午9:46寫道：

> ***@***.**** requested changes on this pull request.
>
> Just make changes to the contacts.txt. In the branch you are working on
> you are making commits to an index.txt and this document will be send up
> when pushing. Delete that file or move it and then commit the changes to
> just the contacts file.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Bora-CS/Batch9-GitPractice/pull/33#pullrequestreview-1295794144>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A44GNLEYDCCI5PMVFOOZVITWXJCNVANCNFSM6AAAAAAUZXW5PE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Commit:
Add my emaill

Commit:
update email
Merge branch 'add-all-names' of github.com:Bora-CS/Batch9-GitPractice into add-all-names

Commit:
add new person

Commit:
other new
